### DISCUSSION TITLE
Armbian_lunar和 Armbian_bookworm怎么启动

### DISCUSSION THREAD

**Discussion by jia070310** [ophub/amlogic-s9xxx-armbian#1516] [Amlogic]
最近看到多了这两个版本，但是按照之前版本的操作方式写入U盘启动不了，直接跳过U盘启动了EMMC。 设备是M401A

**Comment by ophub** [comment]
你试试另外的两个，我估计也U盘启动不了，和系统无关

**Reply by jia070310** [reply]
是的，另外两个可以正常启动。看到新版有点好奇试了试，有问题就反馈出来了

**Comment by ophub** [comment]
这4个的/boot分区的内容完全相同，启动一样，我试了4个都可以。

你可以试试把emmc里的/boot/boot.src改个名，比如boot.src2，这样emmc里面的系统就无法启动了。会从USB启动。

如果想恢复emmc里原来的系统启动，从USB里mount挂载emmc的/boot把名字改回来即可恢复。

**Reply by jia070310** [reply]
解决了，修改EMMC 的boot.src名字就启动了。
